### ISSUE TITLE
TimeoutError: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond

### ISSUE THREAD

**Issue by Monkeyonthecar** [NeuralNine/vidstream#8]
The weird thing is, if my computer streams its screen to the laptop it works fine(on localhost (the computer) it also works fine with itself both ways(server and client)). But when I reverse it (so the laptop streams its screen to the computer) it suddenly doesnt work anymore and i get the folowing error: 

`TimeoutError: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.`

I know this is a very general and non specific error but I still would like to know if more people have problems with the usage of vidstream or have tips.

IMPORTANT NOTE: I did use pyinstaller for my laptop (since it does **not** have python or any packages related to python) to turn it into an exe (with hidden_imports enabled, so that is not the problem). Firewall is also disabled so the problem also doenst lay there. The .exe does work both ways on localhost. Update: after installing python3.9, newest version on pycharm, pyaudio, vidstream on laptop  and just running client/sender.py it still won't send that way but does send it other way. So it has nothing to do with pyinstaller or converting it to exe.

TLDR;
Timeout occurs when client runs vidstream _client.py_  and tries to stream screen to server.But it does work the other way around so that the client runs _server.py_ and server runs _client.py_ (so that the server streams is screen, but the other way is what is needed).

**Comment by Monkeyonthecar** [comment]
nvm found it after days of research and testing. I was using VBOX internal ethernet and not own pc ip adress (allthough all my tcp server client messages did go through). SMH such a dumb mistake
